docs/user/alerting/rule-types.asciidoc

```diff
@@ -15,45 +15,64 @@ see {subscriptions}[the subscription page].
 [[stack-rules]]
 === Stack rules
 
-<<create-and-manage-rules, Stack rules>> are built into {kib}.  To access the *Stack Rules* feature and create and edit rules, users require the `all` privilege. See <<kibana-feature-privileges, feature privileges>> for more information.
+<<create-and-manage-rules, Stack rules>> are built into {kib}. To access the *Stack Rules* feature and create and edit rules, users require the `all` privilege. See <<kibana-feature-privileges, feature privileges>> for more information. In the full rules list in the UI, Stack rules are shown under the proper category `STACK RULES`.
```

Comment:
If you add the intro above, you can remove the line "In the full rules list ..."